labs.yml

```diff
@@ -51,6 +51,17 @@ commands:
       - name: workspace-ids
         description: List of workspace IDs to create account groups from.
 
+  - name: validate
+    is_account_level: true
+    description: |
+      Validate migration readiness across workspaces:
```

Comment:
I'd call it 'feasability' rather than 'readiness'

Reply:
We use 'readiness' more often, e.g. [here](https://github.com/databrickslabs/ucx/blob/cdb7709c3f430a1534321846f1454dc0a91cf74a/src/databricks/labs/ucx/account/aggregate.py#L90)